examples/js/loaders/GLTFLoader.js

```diff
@@ -1370,11 +1550,13 @@ THREE.GLTFLoader = ( function () {
 								? THREE.QuaternionKeyframeTrack
 								: THREE.VectorKeyframeTrack;
 
+							var targetName = node.name ? node.name : node.uuid;
```

Comment:
Ah, that makes sense. Cloning makes the situation complex...
Yeah, let's discuss later.